### PR TITLE
Add primereact w/ npm auto-update

### PR DESCRIPTION
**Pull request by MattIPv4**
Adding primereact using npm auto-update from NPM package primereact.

Resolves https://github.com/cdnjs/cdnjs/issues/12470.

### CODE REVIEW

**Comment by robocdnjs** [comment]
**Tests passed.**
Preview for `packages/p/primereact.json`
Last version (4.2.0):
```
accordion.d.ts
accordion.js
api.d.ts
autocomplete.d.ts
autocomplete.js
breadcrumb.d.ts
breadcrumb.js
button.d.ts
button.js
calendar.d.ts
calendar.js
captcha.d.ts
captcha.js
card.d.ts
card.js
carousel.d.ts
carousel.js
chart.d.ts
chart.js
checkbox.d.ts
checkbox.js
chips.d.ts
chips.js
colorpicker.d.ts
colorpicker.js
column.d.ts
column.js
columngroup.d.ts
columngroup.js
components/accordion/Accordion.css
components/accordion/Accordion.d.ts
components/accordion/Accordion.js
components/autocomplete/AutoComplete.css
components/autocomplete/AutoComplete.d.ts
components/autocomplete/AutoComplete.js
components/autocomplete/AutoCompletePanel.js
components/breadcrumb/BreadCrumb.css
components/breadcrumb/BreadCrumb.d.ts
components/breadcrumb/BreadCrumb.js
components/button/Button.css
components/button/Button.d.ts
components/button/Button.js
components/calendar/Calendar.css
components/calendar/Calendar.d.ts
components/calendar/Calendar.js
components/calendar/CalendarPanel.js
components/captcha/Captcha.d.ts
components/captcha/Captcha.js
components/card/Card.css
components/card/Card.d.ts
components/card/Card.js
components/carousel/Carousel.css
components/carousel/Carousel.d.ts
components/carousel/Carousel.js
components/chart/Chart.css
components/chart/Chart.d.ts
components/chart/Chart.js
components/checkbox/Checkbox.css
components/checkbox/Checkbox.d.ts
components/checkbox/Checkbox.js
components/chips/Chips.css
components/chips/Chips.d.ts
components/chips/Chips.js
components/colorpicker/ColorPicker.css
components/colorpicker/ColorPicker.d.ts
components/colorpicker/ColorPicker.js
components/colorpicker/ColorPickerPanel.d.ts
components/colorpicker/ColorPickerPanel.js
components/column/Column.d.ts
components/column/Column.js
components/columngroup/ColumnGroup.d.ts
components/columngroup/ColumnGroup.js
components/common/Common.css
components/contextmenu/ContextMenu.css
components/contextmenu/ContextMenu.d.ts
components/contextmenu/ContextMenu.js
components/datascroller/DataScroller.css
components/datascroller/DataScroller.d.ts
components/datascroller/DataScroller.js
components/datatable/BodyCell.d.ts
components/datatable/BodyCell.js
components/datatable/BodyRow.d.ts
components/datatable/BodyRow.js
components/datatable/DataTable.css
components/datatable/DataTable.d.ts
components/datatable/DataTable.js
components/datatable/FooterCell.d.ts
components/datatable/FooterCell.js
components/datatable/HeaderCell.d.ts
components/datatable/HeaderCell.js
components/datatable/RowCheckbox.d.ts
components/datatable/RowCheckbox.js
components/datatable/RowRadioButton.d.ts
components/datatable/RowRadioButton.js
components/datatable/RowTogglerButton.d.ts
components/datatable/RowTogglerButton.js
components/datatable/ScrollableView.d.ts
components/datatable/ScrollableView.js
components/datatable/TableBody.d.ts
components/datatable/TableBody.js
components/datatable/TableFooter.d.ts
components/datatable/TableFooter.js
components/datatable/TableHeader.d.ts
components/datatable/TableHeader.js
components/datatable/TableLoadingBody.js
components/dataview/DataView.css
components/dataview/DataView.d.ts
components/dataview/DataView.js
components/deferredcontent/DeferredContent.d.ts
components/deferredcontent/DeferredContent.js
components/dialog/Dialog.css
components/dialog/Dialog.d.ts
components/dialog/Dialog.js
components/dropdown/Dropdown.css
components/dropdown/Dropdown.d.ts
components/dropdown/Dropdown.js
components/dropdown/DropdownItem.d.ts
components/dropdown/DropdownItem.js
components/dropdown/DropdownPanel.js
components/editor/Editor.d.ts
components/editor/Editor.js
components/fieldset/Fieldset.css
components/fieldset/Fieldset.d.ts
components/fieldset/Fieldset.js
components/fileupload/FileUpload.css
components/fileupload/FileUpload.d.ts
components/fileupload/FileUpload.js
components/fullcalendar/FullCalendar.css
components/fullcalendar/FullCalendar.d.ts
components/fullcalendar/FullCalendar.js
components/galleria/Galleria.css
components/galleria/Galleria.d.ts
components/galleria/Galleria.js
components/galleria/GalleriaPreview.d.ts
components/galleria/GalleriaPreview.js
components/galleria/GalleriaThumbnails.d.ts
components/galleria/GalleriaThumbnails.js
components/gmap/GMap.d.ts
components/gmap/GMap.js
components/growl/Growl.css
components/growl/Growl.d.ts
components/growl/Growl.js
components/growl/GrowlMessage.js
components/inplace/Inplace.css
components/inplace/Inplace.d.ts
components/inplace/Inplace.js
components/inputmask/InputMask.d.ts
components/inputmask/InputMask.js
components/inputnumber/InputNumber.css
components/inputnumber/InputNumber.d.ts
components/inputnumber/InputNumber.js
components/inputswitch/InputSwitch.css
components/inputswitch/InputSwitch.d.ts
components/inputswitch/InputSwitch.js
components/inputtext/InputText.css
components/inputtext/InputText.d.ts
components/inputtext/InputText.js
components/inputtextarea/InputTextarea.css
components/inputtextarea/InputTextarea.d.ts
components/inputtextarea/InputTextarea.js
components/keyfilter/KeyFilter.js
components/lightbox/Lightbox.css
components/lightbox/Lightbox.d.ts
components/lightbox/Lightbox.js
components/listbox/ListBox.css
components/listbox/ListBox.d.ts
components/listbox/ListBox.js
components/listbox/ListBoxHeader.d.ts
components/listbox/ListBoxHeader.js
components/listbox/ListBoxItem.d.ts
components/listbox/ListBoxItem.js
components/megamenu/MegaMenu.css
components/megamenu/MegaMenu.d.ts
components/megamenu/MegaMenu.js
components/menu/Menu.css
components/menu/Menu.d.ts
components/menu/Menu.js
components/menubar/Menubar.css
components/menubar/Menubar.d.ts
components/menubar/Menubar.js
components/menubar/MenubarSub.js
components/menuitem/MenuItem.d.ts
components/message/Message.css
components/message/Message.d.ts
components/message/Message.js
components/messages/Messages.css
components/messages/Messages.d.ts
components/messages/Messages.js
components/messages/UIMessage.js
components/multiselect/MultiSelect.css
components/multiselect/MultiSelect.d.ts
components/multiselect/MultiSelect.js
components/multiselect/MultiSelectHeader.js
components/multiselect/MultiSelectItem.js
components/multiselect/MultiSelectPanel.js
components/orderlist/OrderList.css
components/orderlist/OrderList.d.ts
components/orderlist/OrderList.js
components/orderlist/OrderListControls.js
components/orderlist/OrderListSubList.js
components/organizationchart/OrganizationChart.css
components/organizationchart/OrganizationChart.d.ts
components/organizationchart/OrganizationChart.js
components/overlaypanel/OverlayPanel.css
components/overlaypanel/OverlayPanel.d.ts
components/overlaypanel/OverlayPanel.js
components/paginator/CurrentPageReport.js
components/paginator/FirstPageLink.js
components/paginator/LastPageLink.js
components/paginator/NextPageLink.d.ts
components/paginator/NextPageLink.js
components/paginator/PageLinks.js
components/paginator/Paginator.css
components/paginator/Paginator.d.ts
components/paginator/Paginator.js
components/paginator/PrevPageLink.js
components/paginator/RowsPerPageDropdown.js
components/panel/Panel.css
components/panel/Panel.d.ts
components/panel/Panel.js
components/panelmenu/PanelMenu.css
components/panelmenu/PanelMenu.d.ts
components/panelmenu/PanelMenu.js
components/password/Password.css
components/password/Password.d.ts
components/password/Password.js
components/picklist/PickList.css
components/picklist/PickList.d.ts
components/picklist/PickList.js
components/picklist/PickListControls.d.ts
components/picklist/PickListControls.js
components/picklist/PickListItem.d.ts
components/picklist/PickListItem.js
components/picklist/PickListSubList.d.ts
components/picklist/PickListSubList.js
components/picklist/PickListTransferControls.d.ts
components/picklist/PickListTransferControls.js
components/progressbar/ProgressBar.css
components/progressbar/ProgressBar.d.ts
components/progressbar/ProgressBar.js
components/progressspinner/ProgressSpinner.css
components/progressspinner/ProgressSpinner.d.ts
components/progressspinner/ProgressSpinner.js
components/radiobutton/RadioButton.css
components/radiobutton/RadioButton.d.ts
components/radiobutton/RadioButton.js
components/rating/Rating.css
components/rating/Rating.d.ts
components/rating/Rating.js
components/row/Row.d.ts
components/row/Row.js
components/scrollpanel/ScrollPanel.css
components/scrollpanel/ScrollPanel.d.ts
components/scrollpanel/ScrollPanel.js
components/selectbutton/SelectButton.css
components/selectbutton/SelectButton.d.ts
components/selectbutton/SelectButton.js
components/selectbutton/SelectButtonItem.js
components/selectitem/SelectItem.d.ts
components/sidebar/Sidebar.css
components/sidebar/Sidebar.d.ts
components/sidebar/Sidebar.js
components/slidemenu/SlideMenu.css
components/slidemenu/SlideMenu.d.ts
components/slidemenu/SlideMenu.js
components/slider/Slider.css
components/slider/Slider.d.ts
components/slider/Slider.js
components/spinner/Spinner.css
components/spinner/Spinner.d.ts
components/spinner/Spinner.js
components/splitbutton/SplitButton.css
components/splitbutton/SplitButton.d.ts
components/splitbutton/SplitButton.js
components/splitbutton/SplitButtonItem.js
components/splitbutton/SplitButtonPanel.js
components/steps/Steps.css
components/steps/Steps.d.ts
components/steps/Steps.js
components/tabmenu/TabMenu.css
components/tabmenu/TabMenu.d.ts
components/tabmenu/TabMenu.js
components/tabview/TabView.css
components/tabview/TabView.d.ts
components/tabview/TabView.js
components/tieredmenu/TieredMenu.css
components/tieredmenu/TieredMenu.d.ts
components/tieredmenu/TieredMenu.js
components/tieredmenu/TieredMenuSub.js
components/togglebutton/ToggleButton.d.ts
components/togglebutton/ToggleButton.js
components/toolbar/Toolbar.css
components/toolbar/Toolbar.d.ts
components/toolbar/Toolbar.js
components/tooltip/Tooltip.css
components/tooltip/Tooltip.js
components/tooltip/TooltipOptions.d.ts
components/tree/Tree.css
components/tree/Tree.d.ts
components/tree/Tree.js
components/tree/UITreeNode.js
components/treenode/TreeNode.d.ts
components/treetable/TreeTable.css
components/treetable/TreeTable.d.ts
components/treetable/TreeTable.js
components/treetable/TreeTableBody.js
components/treetable/TreeTableBodyCell.js
components/treetable/TreeTableFooter.js
components/treetable/TreeTableHeader.js
components/treetable/TreeTableNode.js
components/treetable/TreeTableRow.js
components/treetable/TreeTableScrollableView.js
components/tristatecheckbox/TriStateCheckbox.d.ts
components/tristatecheckbox/TriStateCheckbox.js
components/util.d.ts
components/utils/DomHandler.js
components/utils/FilterUtils.js
components/utils/ObjectUtils.js
components/utils/UniqueComponentId.js
contextmenu.d.ts
contextmenu.js
datascroller.d.ts
datascroller.js
datatable.d.ts
datatable.js
dataview.d.ts
dataview.js
deferredcontent.d.ts
deferredcontent.js
dialog.d.ts
dialog.js
dropdown.d.ts
dropdown.js
editor.d.ts
editor.js
fieldset.d.ts
fieldset.js
fileupload.d.ts
fileupload.js
fullcalendar.d.ts
fullcalendar.js
galleria.d.ts
galleria.js
gmap.d.ts
gmap.js
growl.d.ts
growl.js
inplace.d.ts
inplace.js
inputmask.d.ts
inputmask.js
inputnumber.d.ts
inputnumber.js
inputswitch.d.ts
inputswitch.js
inputtext.d.ts
inputtext.js
inputtextarea.d.ts
inputtextarea.js
lightbox.d.ts
lightbox.js
listbox.d.ts
listbox.js
megamenu.d.ts
megamenu.js
menu.d.ts
menu.js
menubar.d.ts
menubar.js
message.d.ts
message.js
messages.d.ts
messages.js
multiselect.d.ts
multiselect.js
orderlist.d.ts
orderlist.js
organizationchart.d.ts
organizationchart.js
overlaypanel.d.ts
overlaypanel.js
paginator.d.ts
paginator.js
panel.d.ts
panel.js
panelmenu.d.ts
panelmenu.js
password.d.ts
password.js
picklist.d.ts
picklist.js
progressbar.d.ts
progressbar.js
progressspinner.d.ts
progressspinner.js
radiobutton.d.ts
radiobutton.js
rating.d.ts
rating.js
resources/primereact.css
resources/primereact.min.css
resources/themes/luna-amber/theme.css
resources/themes/luna-blue/theme.css
resources/themes/luna-green/theme.css
resources/themes/luna-pink/theme.css
resources/themes/nova-colored/theme.css
resources/themes/nova-dark/theme.css
resources/themes/nova-light/theme.css
resources/themes/rhea/theme.css
row.d.ts
row.js
scrollpanel.d.ts
scrollpanel.js
selectbutton.d.ts
selectbutton.js
sidebar.d.ts
sidebar.js
slidemenu.d.ts
slidemenu.js
slider.d.ts
slider.js
spinner.d.ts
spinner.js
splitbutton.d.ts
splitbutton.js
steps.d.ts
steps.js
tabmenu.d.ts
tabmenu.js
tabview.d.ts
tabview.js
tieredmenu.d.ts
tieredmenu.js
togglebutton.d.ts
togglebutton.js
toolbar.d.ts
toolbar.js
tooltip.d.ts
tooltip.js
tree.d.ts
tree.js
treetable.d.ts
treetable.js
tristatecheckbox.d.ts
tristatecheckbox.js
```
10 Last version:
```
4.2.0: 445 file(s) matched
4.1.2: 440 file(s) matched
4.1.1: 440 file(s) matched
4.1.0: 440 file(s) matched
4.0.0: 439 file(s) matched
3.4.0: 430 file(s) matched
3.3.3: 428 file(s) matched
3.3.2: 428 file(s) matched
3.3.1: 428 file(s) matched
3.3.0: 426 file(s) matched
```

File: packages/p/primereact.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "primereact",
+  "description": "[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![npm version](https://badge.fury.io/js/primereact.svg)](https://badge.fury.io/js/primereact)",
```
Comment (MattIPv4) [inline packages/p/primereact.json:3]: Write a manual description

File: packages/p/primereact.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "keywords": [],
+  "author": "",
```
Comment (MattIPv4) [inline packages/p/primereact.json:5]: Add manual author

File: packages/p/primereact.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "license": "MIT",
+  "homepage": "http:///primereact/showcase",
```
Comment (MattIPv4) [inline packages/p/primereact.json:7]: Add manual homepage

**Comment by robocdnjs** [comment]
**Tests passed.**
Preview for `packages/p/primereact.json`
Last version (4.2.0):
```
accordion.d.ts
accordion.js
api.d.ts
autocomplete.d.ts
autocomplete.js
breadcrumb.d.ts
breadcrumb.js
button.d.ts
button.js
calendar.d.ts
calendar.js
captcha.d.ts
captcha.js
card.d.ts
card.js
carousel.d.ts
carousel.js
chart.d.ts
chart.js
checkbox.d.ts
checkbox.js
chips.d.ts
chips.js
colorpicker.d.ts
colorpicker.js
column.d.ts
column.js
columngroup.d.ts
columngroup.js
components/accordion/Accordion.css
components/accordion/Accordion.d.ts
components/accordion/Accordion.js
components/autocomplete/AutoComplete.css
components/autocomplete/AutoComplete.d.ts
components/autocomplete/AutoComplete.js
components/autocomplete/AutoCompletePanel.js
components/breadcrumb/BreadCrumb.css
components/breadcrumb/BreadCrumb.d.ts
components/breadcrumb/BreadCrumb.js
components/button/Button.css
components/button/Button.d.ts
components/button/Button.js
components/calendar/Calendar.css
components/calendar/Calendar.d.ts
components/calendar/Calendar.js
components/calendar/CalendarPanel.js
components/captcha/Captcha.d.ts
components/captcha/Captcha.js
components/card/Card.css
components/card/Card.d.ts
components/card/Card.js
components/carousel/Carousel.css
components/carousel/Carousel.d.ts
components/carousel/Carousel.js
components/chart/Chart.css
components/chart/Chart.d.ts
components/chart/Chart.js
components/checkbox/Checkbox.css
components/checkbox/Checkbox.d.ts
components/checkbox/Checkbox.js
components/chips/Chips.css
components/chips/Chips.d.ts
components/chips/Chips.js
components/colorpicker/ColorPicker.css
components/colorpicker/ColorPicker.d.ts
components/colorpicker/ColorPicker.js
components/colorpicker/ColorPickerPanel.d.ts
components/colorpicker/ColorPickerPanel.js
components/column/Column.d.ts
components/column/Column.js
components/columngroup/ColumnGroup.d.ts
components/columngroup/ColumnGroup.js
components/common/Common.css
components/contextmenu/ContextMenu.css
components/contextmenu/ContextMenu.d.ts
components/contextmenu/ContextMenu.js
components/datascroller/DataScroller.css
components/datascroller/DataScroller.d.ts
components/datascroller/DataScroller.js
components/datatable/BodyCell.d.ts
components/datatable/BodyCell.js
components/datatable/BodyRow.d.ts
components/datatable/BodyRow.js
components/datatable/DataTable.css
components/datatable/DataTable.d.ts
components/datatable/DataTable.js
components/datatable/FooterCell.d.ts
components/datatable/FooterCell.js
components/datatable/HeaderCell.d.ts
components/datatable/HeaderCell.js
components/datatable/RowCheckbox.d.ts
components/datatable/RowCheckbox.js
components/datatable/RowRadioButton.d.ts
components/datatable/RowRadioButton.js
components/datatable/RowTogglerButton.d.ts
components/datatable/RowTogglerButton.js
components/datatable/ScrollableView.d.ts
components/datatable/ScrollableView.js
components/datatable/TableBody.d.ts
components/datatable/TableBody.js
components/datatable/TableFooter.d.ts
components/datatable/TableFooter.js
components/datatable/TableHeader.d.ts
components/datatable/TableHeader.js
components/datatable/TableLoadingBody.js
components/dataview/DataView.css
components/dataview/DataView.d.ts
components/dataview/DataView.js
components/deferredcontent/DeferredContent.d.ts
components/deferredcontent/DeferredContent.js
components/dialog/Dialog.css
components/dialog/Dialog.d.ts
components/dialog/Dialog.js
components/dropdown/Dropdown.css
components/dropdown/Dropdown.d.ts
components/dropdown/Dropdown.js
components/dropdown/DropdownItem.d.ts
components/dropdown/DropdownItem.js
components/dropdown/DropdownPanel.js
components/editor/Editor.d.ts
components/editor/Editor.js
components/fieldset/Fieldset.css
components/fieldset/Fieldset.d.ts
components/fieldset/Fieldset.js
components/fileupload/FileUpload.css
components/fileupload/FileUpload.d.ts
components/fileupload/FileUpload.js
components/fullcalendar/FullCalendar.css
components/fullcalendar/FullCalendar.d.ts
components/fullcalendar/FullCalendar.js
components/galleria/Galleria.css
components/galleria/Galleria.d.ts
components/galleria/Galleria.js
components/galleria/GalleriaPreview.d.ts
components/galleria/GalleriaPreview.js
components/galleria/GalleriaThumbnails.d.ts
components/galleria/GalleriaThumbnails.js
components/gmap/GMap.d.ts
components/gmap/GMap.js
components/growl/Growl.css
components/growl/Growl.d.ts
components/growl/Growl.js
components/growl/GrowlMessage.js
components/inplace/Inplace.css
components/inplace/Inplace.d.ts
components/inplace/Inplace.js
components/inputmask/InputMask.d.ts
components/inputmask/InputMask.js
components/inputnumber/InputNumber.css
components/inputnumber/InputNumber.d.ts
components/inputnumber/InputNumber.js
components/inputswitch/InputSwitch.css
components/inputswitch/InputSwitch.d.ts
components/inputswitch/InputSwitch.js
components/inputtext/InputText.css
components/inputtext/InputText.d.ts
components/inputtext/InputText.js
components/inputtextarea/InputTextarea.css
components/inputtextarea/InputTextarea.d.ts
components/inputtextarea/InputTextarea.js
components/keyfilter/KeyFilter.js
components/lightbox/Lightbox.css
components/lightbox/Lightbox.d.ts
components/lightbox/Lightbox.js
components/listbox/ListBox.css
components/listbox/ListBox.d.ts
components/listbox/ListBox.js
components/listbox/ListBoxHeader.d.ts
components/listbox/ListBoxHeader.js
components/listbox/ListBoxItem.d.ts
components/listbox/ListBoxItem.js
components/megamenu/MegaMenu.css
components/megamenu/MegaMenu.d.ts
components/megamenu/MegaMenu.js
components/menu/Menu.css
components/menu/Menu.d.ts
components/menu/Menu.js
components/menubar/Menubar.css
components/menubar/Menubar.d.ts
components/menubar/Menubar.js
components/menubar/MenubarSub.js
components/menuitem/MenuItem.d.ts
components/message/Message.css
components/message/Message.d.ts
components/message/Message.js
components/messages/Messages.css
components/messages/Messages.d.ts
components/messages/Messages.js
components/messages/UIMessage.js
components/multiselect/MultiSelect.css
components/multiselect/MultiSelect.d.ts
components/multiselect/MultiSelect.js
components/multiselect/MultiSelectHeader.js
components/multiselect/MultiSelectItem.js
components/multiselect/MultiSelectPanel.js
components/orderlist/OrderList.css
components/orderlist/OrderList.d.ts
components/orderlist/OrderList.js
components/orderlist/OrderListControls.js
components/orderlist/OrderListSubList.js
components/organizationchart/OrganizationChart.css
components/organizationchart/OrganizationChart.d.ts
components/organizationchart/OrganizationChart.js
components/overlaypanel/OverlayPanel.css
components/overlaypanel/OverlayPanel.d.ts
components/overlaypanel/OverlayPanel.js
components/paginator/CurrentPageReport.js
components/paginator/FirstPageLink.js
components/paginator/LastPageLink.js
components/paginator/NextPageLink.d.ts
components/paginator/NextPageLink.js
components/paginator/PageLinks.js
components/paginator/Paginator.css
components/paginator/Paginator.d.ts
components/paginator/Paginator.js
components/paginator/PrevPageLink.js
components/paginator/RowsPerPageDropdown.js
components/panel/Panel.css
components/panel/Panel.d.ts
components/panel/Panel.js
components/panelmenu/PanelMenu.css
components/panelmenu/PanelMenu.d.ts
components/panelmenu/PanelMenu.js
components/password/Password.css
components/password/Password.d.ts
components/password/Password.js
components/picklist/PickList.css
components/picklist/PickList.d.ts
components/picklist/PickList.js
components/picklist/PickListControls.d.ts
components/picklist/PickListControls.js
components/picklist/PickListItem.d.ts
components/picklist/PickListItem.js
components/picklist/PickListSubList.d.ts
components/picklist/PickListSubList.js
components/picklist/PickListTransferControls.d.ts
components/picklist/PickListTransferControls.js
components/progressbar/ProgressBar.css
components/progressbar/ProgressBar.d.ts
components/progressbar/ProgressBar.js
components/progressspinner/ProgressSpinner.css
components/progressspinner/ProgressSpinner.d.ts
components/progressspinner/ProgressSpinner.js
components/radiobutton/RadioButton.css
components/radiobutton/RadioButton.d.ts
components/radiobutton/RadioButton.js
components/rating/Rating.css
components/rating/Rating.d.ts
components/rating/Rating.js
components/row/Row.d.ts
components/row/Row.js
components/scrollpanel/ScrollPanel.css
components/scrollpanel/ScrollPanel.d.ts
components/scrollpanel/ScrollPanel.js
components/selectbutton/SelectButton.css
components/selectbutton/SelectButton.d.ts
components/selectbutton/SelectButton.js
components/selectbutton/SelectButtonItem.js
components/selectitem/SelectItem.d.ts
components/sidebar/Sidebar.css
components/sidebar/Sidebar.d.ts
components/sidebar/Sidebar.js
components/slidemenu/SlideMenu.css
components/slidemenu/SlideMenu.d.ts
components/slidemenu/SlideMenu.js
components/slider/Slider.css
components/slider/Slider.d.ts
components/slider/Slider.js
components/spinner/Spinner.css
components/spinner/Spinner.d.ts
components/spinner/Spinner.js
components/splitbutton/SplitButton.css
components/splitbutton/SplitButton.d.ts
components/splitbutton/SplitButton.js
components/splitbutton/SplitButtonItem.js
components/splitbutton/SplitButtonPanel.js
components/steps/Steps.css
components/steps/Steps.d.ts
components/steps/Steps.js
components/tabmenu/TabMenu.css
components/tabmenu/TabMenu.d.ts
components/tabmenu/TabMenu.js
components/tabview/TabView.css
components/tabview/TabView.d.ts
components/tabview/TabView.js
components/tieredmenu/TieredMenu.css
components/tieredmenu/TieredMenu.d.ts
components/tieredmenu/TieredMenu.js
components/tieredmenu/TieredMenuSub.js
components/togglebutton/ToggleButton.d.ts
components/togglebutton/ToggleButton.js
components/toolbar/Toolbar.css
components/toolbar/Toolbar.d.ts
components/toolbar/Toolbar.js
components/tooltip/Tooltip.css
components/tooltip/Tooltip.js
components/tooltip/TooltipOptions.d.ts
components/tree/Tree.css
components/tree/Tree.d.ts
components/tree/Tree.js
components/tree/UITreeNode.js
components/treenode/TreeNode.d.ts
components/treetable/TreeTable.css
components/treetable/TreeTable.d.ts
components/treetable/TreeTable.js
components/treetable/TreeTableBody.js
components/treetable/TreeTableBodyCell.js
components/treetable/TreeTableFooter.js
components/treetable/TreeTableHeader.js
components/treetable/TreeTableNode.js
components/treetable/TreeTableRow.js
components/treetable/TreeTableScrollableView.js
components/tristatecheckbox/TriStateCheckbox.d.ts
components/tristatecheckbox/TriStateCheckbox.js
components/util.d.ts
components/utils/DomHandler.js
components/utils/FilterUtils.js
components/utils/ObjectUtils.js
components/utils/UniqueComponentId.js
contextmenu.d.ts
contextmenu.js
datascroller.d.ts
datascroller.js
datatable.d.ts
datatable.js
dataview.d.ts
dataview.js
deferredcontent.d.ts
deferredcontent.js
dialog.d.ts
dialog.js
dropdown.d.ts
dropdown.js
editor.d.ts
editor.js
fieldset.d.ts
fieldset.js
fileupload.d.ts
fileupload.js
fullcalendar.d.ts
fullcalendar.js
galleria.d.ts
galleria.js
gmap.d.ts
gmap.js
growl.d.ts
growl.js
inplace.d.ts
inplace.js
inputmask.d.ts
inputmask.js
inputnumber.d.ts
inputnumber.js
inputswitch.d.ts
inputswitch.js
inputtext.d.ts
inputtext.js
inputtextarea.d.ts
inputtextarea.js
lightbox.d.ts
lightbox.js
listbox.d.ts
listbox.js
megamenu.d.ts
megamenu.js
menu.d.ts
menu.js
menubar.d.ts
menubar.js
message.d.ts
message.js
messages.d.ts
messages.js
multiselect.d.ts
multiselect.js
orderlist.d.ts
orderlist.js
organizationchart.d.ts
organizationchart.js
overlaypanel.d.ts
overlaypanel.js
paginator.d.ts
paginator.js
panel.d.ts
panel.js
panelmenu.d.ts
panelmenu.js
password.d.ts
password.js
picklist.d.ts
picklist.js
progressbar.d.ts
progressbar.js
progressspinner.d.ts
progressspinner.js
radiobutton.d.ts
radiobutton.js
rating.d.ts
rating.js
resources/primereact.css
resources/primereact.min.css
resources/themes/luna-amber/theme.css
resources/themes/luna-blue/theme.css
resources/themes/luna-green/theme.css
resources/themes/luna-pink/theme.css
resources/themes/nova-colored/theme.css
resources/themes/nova-dark/theme.css
resources/themes/nova-light/theme.css
resources/themes/rhea/theme.css
row.d.ts
row.js
scrollpanel.d.ts
scrollpanel.js
selectbutton.d.ts
selectbutton.js
sidebar.d.ts
sidebar.js
slidemenu.d.ts
slidemenu.js
slider.d.ts
slider.js
spinner.d.ts
spinner.js
splitbutton.d.ts
splitbutton.js
steps.d.ts
steps.js
tabmenu.d.ts
tabmenu.js
tabview.d.ts
tabview.js
tieredmenu.d.ts
tieredmenu.js
togglebutton.d.ts
togglebutton.js
toolbar.d.ts
toolbar.js
tooltip.d.ts
tooltip.js
tree.d.ts
tree.js
treetable.d.ts
treetable.js
tristatecheckbox.d.ts
tristatecheckbox.js
```
10 Last version:
```
4.2.0: 445 file(s) matched
4.1.2: 440 file(s) matched
4.1.1: 440 file(s) matched
4.1.0: 440 file(s) matched
4.0.0: 439 file(s) matched
3.4.0: 430 file(s) matched
3.3.3: 428 file(s) matched
3.3.2: 428 file(s) matched
3.3.1: 428 file(s) matched
3.3.0: 426 file(s) matched
```

File: packages/p/primereact.json (hunk 3 above)
Comment (xtuc) [reply packages/p/primereact.json]: bot should ensure that

Comment (MattIPv4) [reply packages/p/primereact.json]: Yeah, just validate that its at least a URL :)

**Pull request merged**